Comparison/src/cz/fidentis/comparison/procrustes/ProcrustesAnalysis.java

```diff
@@ -630,7 +695,7 @@ private float countAxisDistance(float p1, float p2, boolean scaling){
      * @return distance Procrustes distance
      */
     public List<ICPTransformation> doProcrustesAnalysis(ProcrustesAnalysis config2, boolean scaling) {
-        List<ICPTransformation> trans = this.superimpose(config2, scaling);
+        List<ICPTransformation> trans = this.superimposePDM(config2, scaling);

```

Comment:
?? nerozbije to celu prokrustovu analyzu? ta metoda sa pouziva aj mimo vyhladavania landmarkov.